mysql-test/suite/rpl/t/rpl_enable_event_cmd_line_validation.test

```diff
@@ -0,0 +1,94 @@
+# === Purpose ===
+#
+# Test to verify that server config value assigned to --replica-enable-event
+# is properly handled during server startup.
+#
+# === References ===
+#
+# PS-6002: Create/alter event state to keep enabled on replicas
+
+# Test in this file is binlog format agnostic, thus no need
+# to rerun it for every format.
+--source include/big_test.inc
+--source include/have_binlog_format_row.inc
+--source include/master-slave.inc
+--source include/rpl_connection_slave.inc
+
+# Shut down the replica server.
+--let $rpl_server_number = 2
+--source include/rpl_stop_server.inc
+
+# Negative case 1 : Empty
+--let $wild_pattern=
+--let $expect_abort= yes
+--let $reason= Cannot process empty pattern
+--source extra/rpl_tests/rpl_enable_event_cmd_line_validation.inc
+
+# Negative case 2 : No DB
+--let $wild_pattern=.event%
+--let $expect_abort= yes
+--let $reason= The pattern doesn't follow the format <db_pattern>.<event_pattern>
+--source extra/rpl_tests/rpl_enable_event_cmd_line_validation.inc
+
+# Negative case 3 : No Event
+--let $wild_pattern=db%.
+--let $expect_abort= yes
+--let $reason= The pattern doesn't follow the format <db_pattern>.<event_pattern>
+--source extra/rpl_tests/rpl_enable_event_cmd_line_validation.inc
+
+# Negative case 4 : No dot separator
+--let $wild_pattern=event
+--let $expect_abort= yes
+--let $reason= The pattern doesn't follow the format <db_pattern>.<event_pattern>
+--source extra/rpl_tests/rpl_enable_event_cmd_line_validation.inc
+
+# Negative case 5 : Multiple dots
+--let $wild_pattern=db%.event%.random
+--let $expect_abort= yes
+--let $reason= The pattern has multiple dot characters. Please restart with format <db_pattern>.<event_pattern>
+--source extra/rpl_tests/rpl_enable_event_cmd_line_validation.inc
+
+# Negative case 6 : Channel separator
+--let $wild_pattern=channel:db%.event%
+--let $expect_abort= yes
+--let $reason= This server doesn't support per-channel --replica-enable-event feature
+--source extra/rpl_tests/rpl_enable_event_cmd_line_validation.inc
+
+# Positiv case 1 : DB.EVENT
```

Comment:
Here and below "Positive"

Long time no see! :)

Reply:
Laurynas is back!!!